app/services/time_splitter.rb

```diff
@@ -34,17 +39,31 @@ def base_time_entry
   # Line: 26
   # Accessed on: 07.05.2021
   def default_activity(time_entry)
-    time_entry.activity ||= TimeEntryActivity.where(name: 'Development').first
-    time_entry.activity ||= TimeEntryActivity.where(name: 'Entwicklung').first
-    time_entry.activity ||= TimeEntryActivity.default
-    time_entry.activity ||= TimeEntryActivity.where(parent_id: nil, project_id: nil).first
+    possible_activities.each do |activity|
+      time_entry.activity ||= activity unless time_entry.valid?
+    end
+
     time_entry
   end
 
+  def possible_activities
+    [
+      TimeEntryActivity.default,
+      *TimeEntryActivity.where(
+        name: %w[Development Entwicklung]
+      ).or(
+        TimeEntryActivity.where(
+          parent_id: nil,
+          project_id: nil
+        )
+      ).to_a
+    ]
+  end
+
   def create_time_entry(issue, time_entry)
     time_entry.update!(
```

Comment:
Why do stuff in 1 line if you can do it in 4?

Reply:
I don't see how you would write that in one line?

Reply:
```ruby
time_entry.update!(issue: issue, project: issue.project)
```
or is this too long?

I would also add `!` to your function name

Reply:
oh 😆 this line :D true ^^ this could be inlined

Reply:
no.. you only add a ! if there is a "save" version of the method.